packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -76,6 +79,7 @@ async function fundWallet(
     new TextEncoder().encode(
       JSON.stringify({
         destination: walletToFund.classicAddress,
+        xrpAmount: options?.amount,
```

Comment:
What happens if the options pass in an invalid amount? (One that goes beyond how much the Testnet should be able to fund?) - Probably don't need to make it a test, but curious about the edge case inputs.

Reply:
faucet repo has checks on the amounts - if input is invalid it will return an error
https://github.com/xpring-eng/testnet-faucet/blob/336ad485a51fbf63a9d5c335979c68a9886af563/index.js#L135